packages/network/src/Node.js

```diff
@@ -48,4 +48,4 @@ Node.propTypes = {
     handleNodeLeave: PropTypes.func.isRequired,
 }
 
-export default memo(Node)
+export default Node
```

Comment:
Why did you remove the memoization?

Reply:
Not sure... I have put it back and there is no issue 👍